safemode/safemode_dns.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package safemode
+
+import (
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type DNS struct {
```

Comment:
Why add several no-ops safemodes? Do we need to have them?

Reply:
For now, we do not, but Im guessing for the future when more safety nets are considered for each disruption, they would be implemented in these places. Also I wanted to reference them in `safemode.go` but in order for me to reference them they must exist. Its essentially boilerplate for future safety nets, what are your thoughts?